dashboard/app/models/levels/dsl_defined.rb

```diff
@@ -30,6 +30,13 @@
 class DSLDefined < Level
   include Seeded
   after_destroy :delete_level_file
+  validate :validate_level_name
+
+  DEFAULT_LEVEL_NAME = 'unique level name here'
+
+  def validate_level_name
+    errors.add(:name, "is invalid") if name == DEFAULT_LEVEL_NAME
```

Comment:
up to you, but might be helpful here to add a bit more detail. Something like "name cannot be default level name" or something might be a bit clearer?